transit_odp/pipelines/pipelines/dataset_etl/transform.py

```diff
@@ -45,6 +47,19 @@ def transform(self, extracted_data: ExtractedData) -> TransformedData:
         timing_links = extracted_data.timing_links.copy()
         stop_points = extracted_data.stop_points.copy()
         provisional_stops = extracted_data.provisional_stops.copy()
+        flexible_stop_points = self.extracted_data.flexible_stop_points.copy()
+        flexible_timing_links = self.extracted_data.flexible_timing_links.copy()
+        flexible_journey_patterns = self.extracted_data.flexible_journey_patterns.copy()
+        flexible_jp_to_jps = self.extracted_data.flexible_jp_to_jps.copy()
+        flexible_jp_sections = self.extracted_data.flexible_jp_sections.copy()
+
+        # remove flexible stop points from stop points
+        if not flexible_stop_points.empty:
+            # get atco_code value from index
+            flexible_stop_points_list = list(flexible_stop_points.index.values)
+            stop_points = stop_points[
```

Comment:
Would below code return the expected result?
`stop_points.index.difference(flexible_stop_points.index)`

Reply:
done